Cargo.toml

```diff
@@ -256,3 +259,6 @@ harness = false
 [[bench]]
 name = "wasi"
 harness = false
+
+[patch.crates-io]
+wasmprinter = { path = "../wasm-tools/crates/wasmprinter" }
```

Comment:
You already mentioned it but I want to specifically highlight this change as needing to be reverted before this PR can merge.